php/WP_CLI/Runner.php

```diff
@@ -1043,6 +1058,15 @@ public function load_wordpress() {
 
 		// Load Core, mu-plugins, plugins, themes etc.
 		if ( Utils\wp_version_compare( '4.6-alpha-37575', '>=' ) ) {
+			if ( $this->cmd_starts_with( array( 'help' ) ) ) {
+				// Hack: define `WP_DEBUG` and `WP_DEBUG_DISPLAY` to get `wpdb::bail()` to `wp_die()`.
```

Comment:
WP 4.6 and higher, as "wp-settings-cli.php" has the `wbdb->error` check and `wp_die()` after `require_wp_db()`.